src/annotation_processor.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: guide
+title: PDL Schema
+permalink: /annotation_processor
+excerpt: Documentation of using annotation processor in Rest.Li
+---
+
+# Annotation Processor
+This sections introduces a generic schema annotation processing tool in Rest.Li. Before reading this, it is recommended to be familiar with the concept of PathSpec in Rest.Li. [Insert link here]
+
+## Annotating Rest.Li Schemas
+In Rest.Li, schema writers are able to add annotations to schema fields or the schema itself.
+
+**Example**: annotate on the schema fields
+```pdl
+record UserPersonallyIdentifiableInformation {
+  @data_classification = "MEDIUM"
+  firstName: string,
+  @data_classification = "MEDIUM"
+  lastName: string,
+  @data_classification = "LOW"
+  userId: long,
+  @data_classification = "HIGH"
+  socialSecurityNumber: string,
+}
+```
+
+**Example**: annotate directly on the schema
+```pdl
+@data_classification = "HIGH"
+record UserPersonallyIdentifiableInformation {
+  firstName: string,
+  lastName: string,
+  userId: long,
+  socialSecurityNumber: string,
+}
+```
+Note that these annotations are on the schema level.
```

Comment:
Remove this line( redundant) or better say: "The data_classifcation annotation is specified at record level"